cli/archive_test.go

```diff
@@ -0,0 +1,77 @@
+package cli
+
+import (
+	"archive/tar"
+	"bytes"
+	"compress/gzip"
+	"io"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"go.viam.com/test"
+)
+
+func TestGetArchiveFilePaths(t *testing.T) {
+	// Create a temporary directory
+	tempDir := t.TempDir()
+
+	// Create dummy files
+	files := []string{"file1.txt", "file2.txt"}
+	for _, f := range files {
+		fullPath := filepath.Join(tempDir, f)
+		err := os.WriteFile(fullPath, []byte("content"), fs.ModePerm)
+		test.That(t, err, test.ShouldBeNil)
+	}
+
+	// Invoke getArchiveFilePaths function
+	foundFiles, err := getArchiveFilePaths([]string{tempDir})
+	test.That(t, err, test.ShouldBeNil)
+
+	// Validate found files
+	test.That(t, foundFiles, test.ShouldHaveLength, len(files))
+	for _, f := range foundFiles {
+		if _, err := os.Stat(f); os.IsNotExist(err) {
+			t.Errorf("File %s was not found", f)
```

Comment:
```suggestion
			t.Errorf("file %s was not found", f)
```
[nit] test errors should still be lowercase.